readme.md

```diff
@@ -247,50 +189,7 @@ through the entire app. See [Debugging Node.js with Chrome
 DevTools](https://medium.com/@paul_irish/debugging-node-js-nightlies-with-chrome-devtools-7c4a1b95ae27#.59rma3ukm)
 for more info.
 
-## Creating custom audits & gatherers
-
-The audits and gatherers checked into the lighthouse repo are available to any configuration. If you're interested in writing your own audits or gatherers, you can use them with Lighthouse without necessarily contributing upstream.
-
-Better docs coming soon, but in the meantime look at [PR #593](https://github.com/GoogleChrome/lighthouse/pull/593), and the tests [valid-custom-audit.js](https://github.com/GoogleChrome/lighthouse/blob/3f5c43f186495a7f3ecc16c012ab423cd2bac79d/lighthouse-core/test/fixtures/valid-custom-audit.js) and [valid-custom-gatherer.js](https://github.com/GoogleChrome/lighthouse/blob/3f5c43f186495a7f3ecc16c012ab423cd2bac79d/lighthouse-core/test/fixtures/valid-custom-gatherer.js). If you have questions, please file an issue and we'll help out!
-
-> **Tip**: see [Lighthouse Architecture](./docs/architecture.md) for more information on Audits and Gatherers.
-
-### Custom configurations for runs
-
-You can supply your own run configuration to customize what audits you want details on. Copy the [default.js](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/default.js) and start customizing. Then provide to the CLI with `lighthouse --config-path=myconfig.js <url>`
-
-If you are simply adding additional audits/gatherers or tweaking flags, you can extend the default configuration without having to copy the default and maintain it. Passes with the same name will be merged together, all other arrays will be concatenated, and primitive values will override the defaults. See the example below that adds a custom gatherer to the default pass and an audit.
-
-```json
-{
-  "extends": true,
-  "passes": [
-    {
-      "passName": "defaultPass",
-      "gatherers": ["path/to/custom/gatherer.js"]
-    }
-  ],
-  "audits": ["path/to/custom/audit.js"],
-  "aggregations": [
-    {
-      "name": "Custom Section",
-      "description": "Enter description here.",
-      "scored": false,
-      "categorizable": false,
-      "items": [
-        {
-          "name": "My Custom Audits",
-          "audits": {
-            "name-of-custom-audit": {}
-          }
-        }
-      ]
-    }
-  ]
-}
-```
-
-## Tests
+### Tests
 
 Some basic unit tests forked are in `/test` and run via mocha. eslint is also checked for style violations.
```

Comment:
this sentence has morphed into something that makes no sense (and the "basic unit tests" doesn't apply anymore...lots of unit tests plus smoke tests now)